### ISSUE TITLE
owl.carousel.js downloaded from website dont create navs or dots

### ISSUE THREAD

**Issue by arsistyle** [OwlCarousel2/OwlCarousel2#2006]
I downloaded the assets from web ([here](https://github.com/OwlCarousel2/OwlCarousel2/archive/2.2.1.zip)) and the carousel dont create the navs elements, i had download from "view source" from site and that js works.

**Comment by daffzarco** [comment]
I'm also having the problem... js works I can see the nav element but only on mobile

**Comment by keefyhub** [comment]
Can you post your code?

**Comment by arsistyle** [comment]
@keefyhub isnt my code, is the owl.carousel.js, i use the downloaded file from the website (download button) y dont works, but with CTRL+U (in same site) i download the owl.carousel.js from there and works.

(Sorry my english D:)

**Comment by keefyhub** [comment]
do you have an example I can look at?

**Comment by arsistyle** [comment]
no, i dont have an example, but in the DOM dont generate the tags for navs and dots.

**Comment by keefyhub** [comment]
Which file are you using? It should be `/dist/owl.carousel.js` and not `/src/js/owl.carousel.js`

**Comment by iannacone** [comment]
i have the same problem and i tried to reproduce it: [fiddle](https://jsfiddle.net/iannacone/c96pnx5d/5/)

when the number of the items equals to the displayed number of the items, the navs are disabled (and hidden). also if the loop is activated.

try to add another element to the fiddle and you can see that it works normally.
